static-server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+const port = 8080;
+const root = path.join(__dirname, 'candidates');
+const index = path.join(root, 'index.html')
+
+app.use(express.static(root));
+
+app.get('/*', function(request, response) {
+  response.sendFile(path.resolve(index));
+});
```

Comment:
I am not sure but I believe you don't necessarily need this endpoint. You should be able to tweak the `static` middleware in place:
```
app.get('/*', express.static(root));
```